src/Compilers/CSharp/Test/Emit2/Emit/NumericIntPtrTests.cs

```diff
@@ -11546,6 +11519,27 @@ .maxstack  4
 ");
         }
 
+        [Fact, WorkItem(63860, "https://github.com/dotnet/roslyn/issues/63860")]
+        public void IntPtrOperatorOnNewPlatform()
```

Comment:
>NewPlatform [](http://example.com/codeflow?start=36&length=11)

"NewPlatform" is not a specific term because its meaning is going to change as time goes by.